routes/respec/builds/update.ts

```diff
@@ -0,0 +1,52 @@
+import path from "node:path";
+import { mkdir, readFile } from "node:fs/promises";
+
+import type { Request, Response } from "express";
+
+import { env } from "../../../utils/misc.js";
+import sh from "../../../utils/sh.js";
+
+export const PKG_DIR = path.join(env("DATA_DIR"), "respec", "package");
```

Comment:
![a screenshot to show directory structure on server](https://github.com/user-attachments/assets/a081e947-4be9-423e-95e5-e6650fb49217)